liquibase-package-manager.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: liquibase-package-manager
+  version: "0.2.9"
+  epoch: 0
+  description: "Easily manage external dependencies for Database Development. Search for, install, and uninstall liquibase drivers, extensions, and utilities."
+  copyright:
+    - license: Apache-2.0 AND MPL-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/liquibase/liquibase-package-manager
+      tag: v${{package.version}}
+      expected-commit: 1cd39ecef00dee2b6cb800acd2d2410f06179231
+
+  - uses: go/build
+    with:
+      packages: ./cmd/lpm/darwin.go
+      output: lpm
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}/bin"
+      ln -sf /usr/bin/lpm ${{targets.contextdir}}/bin/lpm
+
+  - uses: strip
```

Comment:
nit: we can omit this given go/build pipeline already does the stripping. ref: https://github.com/chainguard-dev/melange/blob/a46420071cfd3beb3a72327bd4254535bc8c43b2/pkg/build/pipelines/go/build.yaml#L58-L62